packages/cardano-services/package.json

```diff
@@ -47,8 +47,9 @@
     "preview:dev": "FILES='-f ../../compose/dev.yml -f dev.yml' yarn preview:up",
     "preview:up": "NETWORK=preview SUBMIT_API_ARGS='--testnet-magic 2' yarn compose:up",
     "preview:down": "NETWORK=preview yarn compose:down",
-    "test": "jest --runInBand -c ./jest.config.js",
+    "test": "jest --runInBand -c ./jest.config.js --selectProjects unit",
     "test:build:verify": "tsc --build ./test",
+    "test:cli": "jest --runInBand -c ./jest.config.js --selectProjects cli",
```

Comment:
Why do we need a separate test script for cli? It should be faster than before now :thinking:

Reply:
It's not faster because before this change lots of option was not tested at all.
We spoken about adding a script which only performs arguments checking, but I abandoned that path because it was not giving the expected improvement: argument parsing (to import options descriptions and default values) already imports more or less the entire SDK: this path was producing code duplication without providing any advantage.